### DISCUSSION TITLE
Visibility of cataloging projects

### DISCUSSION THREAD

**Discussion by HirschSt** [rism-digital/muscat#1342] [Ideas]
We should provide more visibility for grouping sources/sources for research projects / cataloguing projects, etc. Because a project-source/holding relation can be considered as a metadata information, linking could be done in Marc 980. Attribution to a project should be provided afterwards in special cases by maintenance. 

For now we have the following options:

### I.  A dedicated Project authority
This would need a dedicated migration and a redesign of many parts in Muscat, probably the most expensive way to introduce projects.

### II. Project as Institution (aka “Corporate Entity”)
We can use existing institution auth, but then we probably have to extend the model. Also projects can be managed by institutions, but not exist without any relation to institutions or people. Less expensive, but not totally appropriate.

### III. Project as Publication entity
We can use existing publication auth, but then we also probably have to extend the model. A project can release publications, but it looks like that a project is no publication. Less expensive, but not totally appropriate.

### IV. Project as Workgroup
Since every Muscat user is a member of a workgroup, it should be somehow possible to use this workgroup as a metadata for a source/holding etc. In this case also any source update can be triggered. Workgroup model should be improved substantially then. 

Any opinions or additions?

**Comment by HirschSt** [comment]
PS: should we convert this issue to a github discussion? Unfortunately I am not allowed to do this ...

**Reply by ahankinson** [reply]
Done!

**Comment by ahankinson** [comment]
> I. A dedicated Project authority

I think you covered the downsides of this quite well! It would also duplicate quite a lot of functionality we already have, for not much benefit.

> II. Project as Institution (aka “Corporate Entity”)

My preferred option so far! We would probably need to make a new type of institution, and maybe a new relator code for sources, but all the other infrastructure should be the same. 

> III. Project as Publication entity

I think this is currently how we do it, and is an alternative method. But for RISM Online and the OPAC we don't publish the secondary literature as an "authority" (like we do for institutions), so we would need to distinguish between the publications that we are making available (for cataloguing projects) and those which we are only using as a reference. 

> IV. Project as Workgroup

Currently workgroups are private to Muscat. Workgroups are attached to users, and users are attached to sources. We would want to have sources in multiple cataloguing projects (e.g., if a project wants to refer to a source record that it has enhanced significantly, or if it is interested in a group of existing sources in addition to ones it has added.) So this would mean having multiple users per source, which would get very confusing! I think this is my least preferred option.

**Reply by Docudoctor** [reply]
The GND manages projects under institutions.
See the attached screenshot. 
Running projects in the institutions file presumably facilitates data exchange with VIAF, etc.
So I would prefer point II.
Guido

![Projekte](https://user-images.githubusercontent.com/26874482/226655467-01d7bbd8-cb67-41e9-9a0c-ad2b4f1d16ec.PNG)

**Comment by fjorba** [comment]
Maybe you could make use of $b in institutions, now that Muscat can handle them (#1335)
